extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/KnativeConfig.java

```diff
@@ -95,6 +96,12 @@
     @ConfigItem
     Map<String, PortConfig> ports;
 
+    /**
+     * The number of desired pods
+     */
+    @ConfigItem
+    OptionalInt replicas;
```

Comment:
I guess we need to drop this since Knative doesn't seem to have replicas.

Reply:
How would you recommend this? Should I delete the method from the PlatformConfiguration as well? 

Deleting the method would mean that I wouldn't be able use PlatformConfiguration in KubernetesConfigUtil#toMap

Reply:
Yeah, we'll have to remove it from `PlatformConfiguration`.

If we follow the latest suggestion from @iocanel then what you mention won't be a problem since the field won't be used in `toMap`

Reply:
Okey I will remove that form PlatformConfiguration